doc/index.rst

```diff
@@ -1,18 +1,19 @@
 .. _contents:
 
-Overview of autodE_
-=====================
+.. image:: common/logo.png
+   :width: 450
+   :align: center
 
-.. _autodE: https://github.com/duartegroup/autodE
+Overview
+========
 
-**autodE** is a Python module designed for the automated calculation of reaction
-profiles from reactant(s) and product(s) given as 1D SMILES or 3D structures.
-Functionality for conformational sampling (RR algorithm) and transition state
-finding (NEB, CI-NEB, adapt.) is also available.
+.. _autodE: https://github.com/duartegroup/autodE
 
 
-.. warning::
-    **autodE** is still beta release. Please report any bugs `here <https://github.com/duartegroup/autodE/issues>`_
+`autodE <https://github.com/duartegroup/autodE/>`_ is a Python module designed for the automated generation of reaction
+profiles as fast as possible. Profiles are generated using a double-ended search form from reactant(s) and product(s)
```

Comment:
Is this supposed to be "search form from"?

Reply:
no - whoops